txcache/virtualSelectionSession.go

```diff
@@ -1,11 +1,44 @@
 package txcache
 
+import (
+	"math/big"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+)
+
 type virtualSelectionSession struct {
 	session                  SelectionSession
 	virtualAccountsByAddress map[string]*virtualAccountRecord
 }
 
 type virtualAccountRecord struct {
+	initialNonce    core.OptionalUint64
+	initialBalance  *big.Int
```

Comment:
Indeed, good catch! We've previously forgot about it (in the technical document).